persistence-modules/spring-data-jpa-4/src/main/java/com/baeldung/schemageneration/model/Account.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.schemageneration.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "accounts")
+public class Account {
+
+    @Id
+    @GeneratedValue
+    Long id;
```

Comment:
Should these be `private` rather than `package-private`?

Reply:
From "baeldung - for authors - 2 - code in the article" I understood it was advised to use package scope ("use package-scope access modifier if possible").
Nevertheless, I modified to use the **private** access modifier.